docs/hooks/index.md

```diff
@@ -35,64 +27,50 @@ An _Action_ defines one or more _Hooks_ to execute. lakeFS supports three types
 1. Notifying downstream consumers:
    Running a post-merge hook to trigger an Airflow DAG or to send a Webhook to an API, notifying it of the change that happened
 
```

Comment:
Should we add data validations as a use case?

Reply:
sure, feel free, or we can leave it for another PR - the scope of this one is creeping ;)

Reply:
Understood, so lets open a new issue for this